src/sections/SideBar/components/Footer/SideBarFooter.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import { makeStyles, Link, Typography } from "@material-ui/core";
+import { styles } from "../../../../styles/styles";
+import { IconMaker } from '../index';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    justifyContent: "center",
+    flexDirection: "column",
+    width: "100%",
+    minWidth: styles.sizes.sidebarDimensions.minWidth,
+    maxWidth: styles.sizes.sidebarDimensions.maxWidth,
+    backgroundColor: styles.colors.darkGray
+  },
+  container: {
+    display: "flex",
+    justifyContent: "center"
+  },
+  text: {
+    color: "gray",
+    textAlign: "center",
+    paddingBottom: styles.sizes.specialThick.spacingThick2
+  },
+  icon: {
+    color: "gray",
+  },
+  icon2: {
+    color: "white",
+  },
+  link: {
+    margin: styles.sizes.specialThick.spacingThick1,
+    marginTop: styles.sizes.thickness.thick16,
+    marginBottom: styles.sizes.specialThick.spacingThick2,
+    height: styles.sizes.thickness.thick13
+  }
+}));
+
+const Footer = ({ links }) => {
+  const classes = useStyles();
+  const [selectedIndex, setSelectedIndex] = React.useState(0);
```

Comment:
import useState directly from react don't do React.useState. For this do the following
```suggestion
import React,{useState} from "react";
```